versionmd.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"go.coder.com/cli"
+)
+
+var version string
+
+type versioncmd struct {
+	print bool
```

Comment:
I don't think this variable is used

Reply:
I thought that I was going to use it but did not. Will remove it.